rocky/rocky/middleware/onboarding.py

```diff
@@ -9,7 +9,7 @@ def OnboardingMiddleware(get_response):
     def middleware(request):
         response = get_response(request)
         if request.user.is_authenticated:
-            member_onboarded = filter(lambda o: o.onboarded, request.user.organization_members)
+            member_onboarded = OrganizationMember.objects.filter(user=request.user, onboarded=True)
```

Comment:
The old code was there so the cached_property would be used, avoiding a SQL query here. But I'm not sure why it didn't work..